scripts/measures/import-qcdr-measures.js

```diff
@@ -92,7 +87,9 @@ const config = {
  * @param  {object}           config  object defining how to build a new measure from this csv file, including mapping of measure fields to column indices
  * @return {array}            Returns an array of measures objects
  *
- * We trim all data sourced from CSVs because people sometimes unintentionally include spaces or linebreaks
+ * Notes:
+ * 1. The terms 'strata' and 'performance rate' are used interchangeably
```

Comment:
Maybe we should be a little more specific and say that we are referring to the performance rate strata as opposed to the reporting rate strata?

Reply:
Is this clearer or just more confusing? Please feel free to suggest an improved version :)

`The terms [performance rate] 'strata' and 'performance rates' are used interchangeably. `